src/Optimizers/Optimizers.jl

```diff
@@ -3,7 +3,7 @@ abstract type Optimizer end
 function einexpr end
 
 einexpr(T::Type{<:Optimizer}, args...; kwargs...) = einexpr(T(; kwargs...), args...)
-einexpr(config::Optimizer, expr, sizedict) = einexpr(config, SizedEinExpr(expr, sizedict))
+einexpr(config::Optimizer, expr, sizedict) = sum(einexpr.((config,), [SizedEinExpr(exp, sizedict) for exp in [comp for comp in components(expr)]]))
```

Comment:
```suggestion
einexpr(config::Optimizer, expr, sizedict) = sum(einexpr.((config,), map(exp -> SizedEinExpr(exp, sizedict), components(expr))))
```

Maybe it could be rewritten like this?

Reply:
Yeah, I completely agree. I guess I am still getting used to Julia haha
Made the change! Thanks